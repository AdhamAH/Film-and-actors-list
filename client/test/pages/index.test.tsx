import React from 'react'
import { render } from '../testUtils'
import Home from '../../pages'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <Home
        data={{
          getAllFilms: [
            {
              id: 48,
              title: 'film 2',
              playTime: '2021-12-14T10:51:13.000Z',
              createdAt: '2021-12-14T15:57:36.000Z',
              creatorUID: 2,
              updatedAt: '2021-12-14T15:57:36.000Z',
              user: {
                username: 'dude',
                createdAt: '2020-12-14T15:57:36.000Z',
                updatedAt: '2020-12-14T15:57:36.000Z',
                email: 'dude@dude.com',
                id: 2,
              },
            },
          ],
        }}
      />,
      {}
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
