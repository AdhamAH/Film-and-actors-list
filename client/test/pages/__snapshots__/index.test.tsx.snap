// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <button
      class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
      tabindex="0"
      type="button"
    >
      create post
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
    <main>
      <div
        class="MuiBox-root css-1033rfx"
      >
        <p>
          film 2
        </p>
      </div>
    </main>
  </div>
</DocumentFragment>
`;
